Tests/UnitTests/RequestCoordinatorTests.swift

```diff
@@ -245,5 +245,88 @@ class RequestCoordinatorTests: XCTestCase {
         
         self.wait(for: [expectation], timeout: 2.0)
     }
+    
+    func testEnqueueRetriableRequest_backsOff_succeeds() {
```

Comment:
We're still missing a test that verified the backoff calculation does what we expect. The retry could be retrying in a loop (i.e. no backoff) and this test would still pass.